src/components/avatar/Avatar.scss

```diff
@@ -22,27 +29,20 @@
         width: 100%;
     }
 
-    .avatar-icon,
-    .avatar-image {
-        @extend %circle;
+    img.avatar-image {
+        image-rendering: optimizeSpeed;
```

Comment:
Deprecated according to MDN https://developer.mozilla.org/en-US/docs/Web/CSS/image-rendering#Browser_compatibility